### PR TITLE
testing

### PR DESCRIPTION
**Pull request by JustD4nTe**

### CODE REVIEW

**Review by upsu90 — APPROVED** [review]

**Comment by Kwolik** [comment]
ok

**Comment by KamilDonda** [comment]
testowy komentarz

File: index.html [hunk 1]
```diff
@@ -1,8 +1,14 @@
...
     <head></head>
     <body>
+        <article>*** **</article>
         <button>asfsdf</button>
-        <p>asdfsadfkjlsadf</p>
-		<span> Robert Kwoll jestem idiota bo nie znam githuba es </span>
+
+        <article>*** **</article>
+        <p>asdfasdfs</p>
+        <p>asdfasdfs</p>
```
Comment (JustD4nTe) [inline index.html:9]: sdfasdfasdf

File: index.html [hunk 2]
```diff
@@ -1,8 +1,14 @@
 <html>
     <head></head>
     <body>
+        <article>*** **</article>
         <button>asfsdf</button>
-        <p>asdfsadfkjlsadf</p>
-		<span> Robert Kwoll jestem idiota bo nie znam githuba es </span>
+
+        <article>*** **</article>
```
Comment (JustD4nTe) [inline index.html:7]: sdfgsdfgsdfbcxvb

**Pull request merged**